packages/image_picker/image_picker_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.2.0
+
+**breaking changes**:
+- pickImage now returns XFile instead of PickedFile
+- pickVideo now returns XFile instead of PickedFile
+- changed LostData file parameter type to XFile
+
```

Comment:
This also needs to remove the PickedFile class from this package, and its tests. That way we only use XFile from this major version on. I'll let the plugins team know tomorrow at standup that this major version change is incoming.